db/models.py

```diff
@@ -1,8 +1,18 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.db import transaction
+from django.utils import timezone
+from typing import Any
+
+
+class User(AbstractUser):
```

Comment:
Code Style: You've imported `get_user_model` but you're still using `AbstractUser` for your User model. It's recommended to use `get_user_model()` instead. This allows you to work with the active user model in Django, whether it's the built-in User model or a custom one.